.github/workflows/deploy.yml

```diff
@@ -73,12 +73,17 @@ jobs:
       - run: |
           cat deployments/templates/deployment.yml | envsubst > deployments/deployment.yml
           cat deployments/templates/ingress.yml | envsubst > deployments/ingress.yml
+          cat deployments/templates/network-monitor.yml | envsubst > deployments/network-monitor.yml
+          cat deployments/templates/prometheus-rules.yml | envsubst > deployments/prometheus-rules.yml
           cat deployments/templates/service.yml | envsubst > deployments/service.yml
+          cat deployments/templates/service-monitor.yml | envsubst > deployments/service-monitor.yml
         env:
           IMAGE_TAG: ${{ github.sha }}
           REGISTRY: ${{ steps.login-ecr.outputs.registry }}
           REPOSITORY: ${{ vars.ECR_REPOSITORY }}
           NAMESPACE: ${{ secrets.KUBE_NAMESPACE }}
+          ALERT_SEVERITY: laa-ccw-fees-data-dev
```

Comment:
The `dev` environment is hardcode here. I am planning to raise a separate ticket to refactor the deployment to have a deploy to dev step, where the environment name is passed a parameter. To make it easier to set up deployment for further environments.